test/cpp/qps/server_async.cc

```diff
@@ -172,8 +172,9 @@ class AsyncQpsServerTest final : public grpc::testing::Server {
     for (auto cq = srv_cqs_.begin(); cq != srv_cqs_.end(); ++cq) {
       bool ok;
       void* got_tag;
-      while ((*cq)->Next(&got_tag, &ok))
+      while ((*cq)->Next(&got_tag, &ok)) {
         ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done